app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Name: <%= @customer.name %></h1>
+
+<h2>"Items:"</h2>
+
+<% @items.each do |item| %>
+<p><%= item.name %></p>
+<p><%= item.price %></p>
+<p><%= item.supermarket.name %></p>
+<% end %>
```

Comment:
Make sure to indent the code inside the block, so lines 6-8 should be tabbed over one. 
Good job iterating over the items, this looks good.